src/test/groovy/steps/DataAnalyticsSteps.groovy

```diff
@@ -181,34 +177,34 @@ class DataAnalyticsSteps {
     def xray409(count, calls){
         while (count <= calls) {
             def policyName = "new-policy"
-            xraySteps.createPolicy(policyName, username, password, xrayBaseUrl)
+            createPolicy(policyName)
             count++
         }
     }
 
     def xray500(count, calls){
         while (count <= calls) {
             def policyName = "non-existing-policy"
-            Response policy = xraySteps.getPolicy(policyName, username, password, xrayBaseUrl)
+            Response policy = xraySteps.getPolicy(policyName, username, password, xrayURL)
```

Comment:
This should be curried as well: 
```groovy
def getPolicy = (xraySteps.&getPolicy).rcurry(username,password,xrayURL)
```
sample:
```
groovy:000> thing = {int a, int b, int c -> 
groovy:001> println(a)
groovy:002> println(b)
groovy:003> println(c)
groovy:004> (a/b) + c
groovy:005> }
===> groovysh_evaluate$_run_closure1@52b06bef
groovy:000> plusFive = thing.rcurry(5,1)
===> org.codehaus.groovy.runtime.CurriedClosure@5b275811
groovy:000> plusFive(15)
15
5
1
===> 4
```
https://groovy-lang.org/closures.html#_right_currying
https://blog.mrhaki.com/search?q=rcurry